less/navigation.less

```diff
@@ -1 +1,48 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+
+.nav-bar {
+    display: flex;
+    flex-direction: column;
+  background: #f4f4f4;
+  padding: 20px 0;
+  border: 1px double #e0e0e0;
+  align-items: center;
+  border: red;
+
+
+  nav {
+    // border-color: red;
+    // border: 1px solid red;
+    display: flex;
+    justify-content: space-evenly;
+  
+    a {
+      text-decoration: none;
+      font-size: 2rem;
+      color: #22283B;
```

Comment:
it would make sense to put this in the variables.less file and then come up with a cool name for it and use it here.